mewarpx/tests/test_capacitive_discharge.py

```diff
@@ -0,0 +1,366 @@
+    #         ],
+    #         scraper=run.scraper,
+    #         diag_steps=run.diag_steps,
+    #         linres=50,
+    #         jslist=0,
+    #         name='particle_histogramcurrent'
+    #     )
+
+    # warputil.warp_generate()
+
+    # # Run the main WARP loop
+    # while not run.runresults.terminate_flag:
+    #     warp.step(run.diag_steps)
+
+    #######################################################################
+    # Cleanup and final output                                            #
+    #######################################################################
+
+    # run.runresults.finalize_save()
+
+    # out, _ = capsys.readouterr()
+
+    # filelist = [
+    #     "diags/fields/Barrier_index_0000000000.pdf",
+    #     "diags/fields/Barrier_index_0000000000.png",
+    #     "diags/fields/Barrier_index_0000000106.pdf",
+    #     "diags/fields/Barrier_index_0000000106.png",
+    #     "diags/fields/Electric_field_strength_0000000000.npy",
+    #     "diags/fields/Electric_field_strength_0000000000.pdf",
+    #     "diags/fields/Electric_field_strength_0000000000.png",
+    #     "diags/fields/Electric_field_strength_0000000106.npy",
+    #     "diags/fields/Electric_field_strength_0000000106.pdf",
+    #     "diags/fields/Electric_field_strength_0000000106.png",
+    #     "diags/fields/Electrostatic_potential_0000000000.npy",
+    #     "diags/fields/Electrostatic_potential_0000000000.pdf",
+    #     "diags/fields/Electrostatic_potential_0000000000.png",
+    #     "diags/fields/Electrostatic_potential_0000000106.npy",
+    #     "diags/fields/Electrostatic_potential_0000000106.pdf",
+    #     "diags/fields/Electrostatic_potential_0000000106.png",
+    #     "diags/fields/Net_charge_density_0000000000.npy",
+    #     "diags/fields/Net_charge_density_0000000106.npy",
+    #     "diags/fields/Net_charge_density_0000000106.pdf",
+    #     "diags/fields/Net_charge_density_0000000106.png",
+    #     "diags/traces/trace_0000000106.npz",
+    #     "diags/xzsolver/hdf5/data0000000106.h5",
+    #     "diags/fluxes/anode_plane_scraped.csv",
+    #     "diags/fluxes/cathode_scraped.csv",
+    #     "diags/results.dpkl",
+    #     "diags/results.txt",
+    #     "diags/runinfo.dpkl",
+    # ]
+
+    # if use_2d_scraper:
+    #     filelist += [
+    #         "diags/histograms/particle_histogram_0000000106.npy",
+    #         "diags/histograms/particle_histogramcurrent_0000000106.npy",
+    #         "diags/histograms/particle_histogramcurrent_setup.p",
+    #         "diags/histograms/particle_histogramcurrent_zold_0000000106.pdf",
+    #         "diags/histograms/particle_histogram_setup.p",
+    #         "diags/histograms/particle_histogram_uxbirth_0000000106.pdf",
+    #         "diags/histograms/particle_histogram_uxbirth_uzbirth_0000000106.pdf",
+    #         "diags/histograms/particle_histogram_uzbirth_0000000106.pdf",
+    #     ]
+
+    # print(out)
```

Comment:
Well you could keep some asserts like this that checks a few lines in the output are in fact present. ANd if there are output files that should be created you can check for them.